src/devtools/client/debugger/src/components/shared/Accordion.css

```diff
@@ -13,6 +13,11 @@
   height: 100%;
 }
 
+.accordion > li {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
Is this doing anything?

Reply:
Yeah — without this, the entire Events panel (header+content/events) overflows the sidebar container.

Reply:
Hmmmmmmm maybe we should look at this together. I'm not seeing that.

Reply:
Ah, I see the problem. It's because we have overflow-hidden elsewhere. Well, let's say this is fine for now, I'll fix it up as part of my accordion CSS overhaul.